maa.jwt.verifier/include/context.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <string>
+#include <vector>
+#include <iostream>
+
+namespace mvj {
```

Comment:
what does mvj stand for?

Reply:
the tool, when originally created, used to be something like maa-validate-jwt. I'll update it to avoid confusion